pkg/genericapiserver/server/openapi/openapi.go

```diff
@@ -33,13 +33,15 @@ import (
 
 const (
 	OpenAPIVersion = "2.0"
+	ExtensionTags  = "x-kubernetes-tags"
```

Comment:
Sure, I am going to use x-kubernetes-gvk and make the value a little more descriptive.